### PR TITLE
Modify rule S5334: Change text to the education framework format [Python][APPSEC-234]

### PR DESCRIPTION
**Pull request by gaetan-ferry-sonarsource**

### CODE REVIEW

**Comment by gaetan-ferry-sonarsource** [comment]
The _How to fix it_ section uses two common files created in https://github.com/SonarSource/rspec/pull/1383.

File: rules/S5334/python/how-to-fix-it/python.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+==== Noncompliant code example
+
+[source,python,diff-id=1,diff-type=noncompliant]
+----
+@app.route("/")
+def example():
+    operation = request.args.get("operation")
+    eval(f"product_{operation}()") # Noncompliant
+    return "OK"
```
Comment (pierre-loup-tristant-sonarsource) [inline rules/S5334/python/how-to-fix-it/python.adoc:14]: I suggest adding the import to make it clear that `request` is actually the `flask.request` global object.

```suggestion
from flask import request

@app.route("/")
def example():
    operation = request.args.get("operation")
    eval(f"product_{operation}()") # Noncompliant
    return "OK"
```

File: rules/S5334/python/how-to-fix-it/python.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+[source,python,diff-id=1,diff-type=compliant]
+----
+@app.route("/")
+def example():
+    allowed = ["add", "remove", "update"]
+    operation = allowed[request.args.get("operationId")]
+    eval(f"product_{operation}()")
+
+    return "OK"
```
Comment (pierre-loup-tristant-sonarsource) [inline rules/S5334/python/how-to-fix-it/python.adoc:27]:
```suggestion
from flask import request

@app.route("/")
def example():
    allowed = ["add", "remove", "update"]
    operation = allowed[request.args.get("operationId")]
    eval(f"product_{operation}()")

    return "OK"
```

**Review by pierre-loup-tristant-sonarsource — APPROVED** [review]
LGTM. 
Please look at the suggestion I made before merging.
Also, since https://github.com/SonarSource/rspec/pull/1383 was merged, please update this PR (merge or rebase) and check that the validate_asciidoc check passes correctly before merging to [feature/education](https://github.com/SonarSource/rspec/tree/feature/education).

**Pull request merged**